easee/easee.py

```diff
@@ -24,8 +24,26 @@ async def raise_for_status(response):
             e.message = str(data)
         else:
             data = await response.text()
-        _LOGGER.error("Error in request to Easee API: %s", data)
-        raise Exception(data) from e
+
+        if 400 == response.status:
+            _LOGGER.error(
+                "Bad request service " + f"({response.status}: {data} {response.url})"
+            )
+        elif 403 == response.status:
+            _LOGGER.error(
+                "Forbidden service " + f"({response.status}: {data} {response.url})"
+            )
+        elif 404 == response.status:
+            _LOGGER.error(
+                "Service not found " + f"({response.status}: {data} {response.url})"
+            )
```

Comment:
Does these occur in normal use of the library or are they bcos of faulty code?

Reply:
Not due to faulty code. I think I've received all of them when code is good, but return is unexpected.

403 I recall getting with correct code but since the API didn't allow the specific function (think it was setRatedCurrent). The call is documented in API but I guess it is restricted since electrical regulations require preventing the user to fiddle with ratings. It is however possible to do it after a reboot through local access, assume the API also works in that mode.

400 and/or 404 I believe I got too. Not sure which one, but got one of them when requesting session energy which didn't exist since I didn't charge (i.e. empty). I hope they plan to improve the api returning just an emtpy list or something, but if I'm not mistaken it doesn't today. Perhaps a bug on their side, not sure. I also got some funny reply on start/stop when car not disconnected, but this I need to investigate a bit more and perhaps we should do some "if connected" or similar to improve code.

Reply:
I guess 404 is possible to get with faulty code too though, but only due to wrong spelling of api calls and our sensors, not invalid code per se.

Reply:
Very worrying of the API returns 404 and 403 for common api calls that sometimes work. We will need to keep an eye on this bcos it is not good. If it happens again we should report it to Easee

Reply:
Yes, I was also kindof surprised on some of the returns. But let's keep an eye on it.